libraries/bot-ai-qna/src/main/java/com/microsoft/bot/ai/qna/dialogs/QnAMakerDialog.java

```diff
@@ -938,11 +945,13 @@ private static void resetOptions(DialogContext dc, QnAMakerDialogOptions dialogO
                 // Call Active Learning Train API
                 return this.getQnAMakerClient(stepContext).thenCompose(qnaClient -> {
                     try {
-                        return qnaClient.callTrain(feedbackRecords);
+                        return qnaClient.callTrain(feedbackRecords).thenCompose(task ->
+                            stepContext.next(new ArrayList<QueryResult>().add(qnaResult)));
                     } catch (IOException e) {
-                        LoggerFactory.getLogger(QnAMakerDialog.class).error("callTrain");
+                        e.printStackTrace();
```

Comment:
Not quite sure about this, check another examples of the repository how this is managed